src/util/decision_procedure.h

```diff
@@ -12,11 +12,12 @@ Author: Daniel Kroening, kroening@kroening.com
 #ifndef CPROVER_UTIL_DECISION_PROCEDURE_H
 #define CPROVER_UTIL_DECISION_PROCEDURE_H
 
-#include "message.h"
+#include <iosfwd>
+#include <string>
```

Comment:
Not sure why these are needed.

Reply:
`decision_procedure_text` returns a `std::string` and `print_assignment` takes a `std::ostream` as parameter. (Previously both includes were dragged in by `message.h`.)